go.mod

```diff
@@ -18,8 +18,61 @@ require (
 	google.golang.org/appengine v1.6.6 // indirect
 	google.golang.org/genproto v0.0.0-20200626011028-ee7919e894b5 // indirect
 	google.golang.org/grpc v1.30.0
-	google.golang.org/protobuf v1.25.0 // indirect
-	k8s.io/apimachinery v0.18.5
+	google.golang.org/protobuf v1.25.0
+	k8s.io/apimachinery v0.18.0
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/klog v1.0.0
+	k8s.io/kubernetes v1.18.0
```

Comment:
any reason why 1.18 and not something newer?

Reply:
no specific reason. took pdcsi driver as a reference. Will clean it up to newer version in a separate patch